test/burn/org/apache/cassandra/utils/memory/LongBufferPoolTest.java

```diff
@@ -230,6 +272,10 @@ else if (!recycleFromNeighbour())
                         }
                     }
 
+                    if (currentTargetSize == 0) {
```

Comment:
According to the code style, you can skip braces here. We also follow follow Sun Style so braces go on new lines.
Reference: http://cassandra.apache.org/doc/latest/development/code_style.html

Reply:
fixed